drivers/hmis/app/models/hmis/hud/processors/current_living_situation_processor.rb

```diff
@@ -22,5 +22,15 @@ def schema
     def dependent_destroyable?
       true
     end
+
+    def assign_metadata
+      super
+
+      current_living_situation = @processor.send(factory_name, create: false)
+      return unless current_living_situation.verified_by_project_id
+
+      project_name = Hmis::Hud::Project.find(current_living_situation.verified_by_project_id)&.name
+      current_living_situation.verified_by = project_name
```

Comment:
probably worth a comment here, similar to the one in the form structure, explaining that we copy the name into the HUD field `VerifiedBy`

Reply:
Calling out that the length of these two fields `VerifiedBy` and `ProjectName` are not equivalent in the spec, so this could be problematic to store project name, or at least to assume it will match on imported data.

Reply:
Thanks, that's a good catch. It looks like we actually have that constraint on the database -- max 100 chars on VerifiedBy. So, we should truncate here if it's longer than 100.

It's a little odd to me that we put that constraint on the DB as opposed to just truncating when we generate CSVs. But that's another question probably for the bigger db-conforming-to-csv-spec convo.